src/test/integration/article-responsive-display.test.ts

```
@@ -0,0 +1,100 @@
+import { describe, it, expect } from 'vitest';
+import { readFileSync } from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)